proprietary/Components/src/utrecht/form-field.tokens.json

```diff
@@ -0,0 +1,17 @@
+{
+  "utrecht": {
+    "form-field": {
+      "margin-block-start": { "value": "{denhaag.space.block.lg}" },
+      "margin-block-end": { "value": "{denhaag.space.block.lg}" },
+      "label": {
+        "margin-block-end": { "value": "{denhaag.space.block.2xs}" }
+      },
+      "description": {
+        "color": { "value": "{denhaag.color.grey.4}" },
+        "font-family": { "value": "{utrecht.document.font-family}" },
+        "font-size": { "value": "{denhaag.typography.scale.s.font-size}" },
+        "font-weight": { "value": "{denhaag.typography.weight.regular}" }
+      }
```

Comment:
https://github.com/nl-design-system/denhaag/pull/1083#discussion_r970749799

The `description` tokens should still be moved to `form-field-description.tokens.json`.